### ISSUE TITLE
"Font Forge" not found when running webfont.

### ISSUE THREAD

**Issue by mahimaSabharwal** [sapegin/grunt-webfont#247]
Hi,

i am getting this error when running webfont task. I've followed all installation instructions as specified. I am a windows user, so i just installed grunt plugin for webfont. Still getting this error.

Need urgent help for this.

![image](https://cloud.githubusercontent.com/assets/12639040/7983800/b9250c38-0ae1-11e5-9635-0187c94dd806.png)

**Comment by brunano21** [comment]
Hi,
Have a look to the documentation, please.
Here [Windows installation](https://github.com/sapegin/grunt-webfont#windows) and here [engine](https://github.com/sapegin/grunt-webfont#engine)

Since you are on a Win machine, you'll need to set the `engine` variable to `node`, because the default value is `font-forge`.

So please, use: `engine: 'node'`,

**Comment by mahimaSabharwal** [comment]
Thanks for the reply.

I've tried using node engine too. But the icons that were generated by the engine were very small in dimensions. 1x1 i guess.

I also tried using other configurations like fontHeight, but results were null. Thats why i had to switch back to fontforge.

webfont: {
            icons: {
                src: 'src/svg/*.svg',
                dest: 'src/fonts',
                options: {
                    font:"icons",
                    hashes: false
                  }
                }
            }


Webfont gives a lot of configurations which makes it useful for me. If i can use fontforge with the same, that would help me a lot.
